exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "exercise": "reverse-string",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "detects reverse-string",
+      "property": "reverse-string",
+      "input": "robot", "superman", "ramen",
```

Comment:
This is not valid JSON.

Also I can't find a word in the desription.md that tells me that I have to handle more than a single string.

And it feels weird that this is the sole testcase…

I'd expect to have at least 3 testcases:

1. reverse the empty string
2. reverse a string consisting of a single lowercase letter
3. reverse a string having more than a single lowercase letter
4. reverse a string with at least a capital letter
5. reverse a string that contains a proper sentence with punctuation.
6. reverse a string twice and check if it is the original again (add a comment that this is perfect for poperty based testing)